docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -241,7 +241,19 @@ The default [Strapi theme](https://github.com/strapi/design-system/tree/main/pac
 
 ### WYSIWYG editor
 
-To change the current WYSIWYG, you can either install a third-party plugin, or take advantage of the bootstrap lifecycle (see [Admin Panel API](/developer-docs/latest/developer-resources/plugin-api-reference/admin-panel.md#bootstrap)).
+To change the current WYSIWYG, you can install a [third-party plugin](https://market.strapi.io/), create your own plugin (see [creating a new field in the admin panel](/developer-docs/latest/guides/registering-a-field-in-admin.md)) or take advantage of the [bootstrap lifecycle](/developer-docs/latest/developer-resources/plugin-api-reference/admin-panel.md#bootstrap):
```

Comment:
This text is not part of the _migration_ process,
but I thought it would be a good idea to be more explicit here, especially since Julie updated the guide. 
Lots of people are asking about how to customize the WYSIWYG and this piece of documentation should give them all the answers 😄 
WDYT @soupette ?

Reply:
Yes that's a great idea!